package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "jquery-ajax-unobtrusive",
+  "version": "3.2.5",
+  "description": "Add-on to jQuery Ajax to enable unobtrusive options in data-* attributes",
+  "main": "jquery.unobtrusive-ajax.js",
+  "repository": {
+    "type": "git",
+    "url": "http://github.com/aspnet/jquery-ajax-unobtrusive"
+  },
+  "keywords": [
+    "unobtrusive",
+    "ajax",
+    "mvc",
+    "asp.net",
+    "jquery"
+  ],
+  "author": "Microsoft",
+  "license": "https://www.microsoft.com/web/webpi/eula/aspnetcomponent_rtw_enu.htm",
```

Comment:
I picked up the license details link from the existing nuget package available on nuget.org. Sure will change it.

You mean updating the JS files' headers. How should the content look like? should it be the standard .NET foundation header?